From 5e71f6eb17cbb119f0f995f234bf1887a4f77058 Mon Sep 17 00:00:00 2001
From: Benjamin Marzinski <bmarzins@redhat.com>
Date: Thu, 11 Oct 2018 15:43:47 -0500
Subject: [PATCH] libmultipath: free allocated value in set_int

set_int() needs to free the buffer it got back from set_value()

Fixes: 8f9ac30d ("libmultipath: fix set_int error path")
Signed-off-by: Benjamin Marzinski <bmarzins@redhat.com>
---
 libmultipath/dict.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libmultipath/dict.c b/libmultipath/dict.c
index bf4701ee..c3f5a6e6 100644
--- a/libmultipath/dict.c
+++ b/libmultipath/dict.c
@@ -39,6 +39,7 @@ set_int(vector strvec, void *ptr)
 
 	*int_ptr = atoi(buff);
 
+	FREE(buff);
 	return 0;
 }
 
-- 
2.11.0

